src/Learning/KWData/KWClass.cpp

```diff
@@ -2067,28 +2109,90 @@ boolean KWClass::CheckNameWithMessage(const ALString& sValue, int nEntity, ALStr
 
 boolean KWClass::CheckLabelWithMessage(const ALString& sValue, int nEntity, ALString& sMessage)
 {
+	const int nMaxLabelLength = 100000;
```

Comment:
Could we group these constants (also below) in a header file like `KWClass.h` (AFAIU we can do this in C++, as they default to static linkage, so there is no name clash with other translation units)?